codex-process-data-transfer/src/main/java/de/netzwerk_universitaetsmedizin/codex/processes/data_transfer/validation/ValidationPackage.java

```diff
@@ -163,8 +163,10 @@ private Consumer<ValidationPackageEntry> doParseResources(FhirContext context, L
 
 			try
 			{
-				IBaseResource resource = context.newJsonParser()
-						.parseResource(new String(entry.getContent(), StandardCharsets.UTF_8));
+				String resourceString = new String(entry.getContent(), StandardCharsets.UTF_8);
+				// fix profiles because their text contains invalid html
```

Comment:
Is there a GitHub Issue we could reference here?

Reply:
added